.env.development

```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_ENV_DEKORATOR=dev
```

Comment:
Topp å erstatte med env.development, men da trenger vi også f.eks. env.production. Problemet for oss er at env.development ikke gjenspeiler vårt dev miljø men vårt lokale miljø og env.production gjenspeiler vårt dev miljø og antagelighvis vårt prod miljø. Så vi må finne på noe smart her tror jeg :)

Reply:
.env.development er som jeg tolket dokumentasjonen kun for "local development", altså når man kjører npm run dev. Side både Dev og prod kjører "produksjons-bygg" av appen må vi løse dynamiske variabler på en eller annen måte 🤔

Reply:
Ja. Er foreløpig usikker hvordan vi kan løse det. Men har en mistanke om at vi kanskje kan definere opp et nytt miljø og konfigurere dette opp under scripts i package.json i løsningen